### PR TITLE
FORMS-7603 forms core components changes

### PR DESCRIPTION
**Pull request by barshat7**
### Change Includes

- Update core component (6.5) versions to 1.1.18
- Unhide "Adaptive Forms Container" component
- Rename "AEM Forms Container" to "Adaptive Form - Embed" 
- Header Image not being copied to form editor from template editor - Fix
- Test Cases For archetype 650

## Related Issue
FORMS-7603

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/main/archetype/pom.xml [hunk 1]
```diff
@@ -83,8 +83,8 @@
 
 #if ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y")
 #if ($aemVersion == "cloud")
-        <core.forms.components.version>2.0.14</core.forms.components.version>
-        <core.forms.components.af.version>2.0.14</core.forms.components.af.version>
+        <core.forms.components.version>2.0.16</core.forms.components.version>
```
Comment (rismehta) [inline src/main/archetype/pom.xml]: Last released version is 2.0.14. https://github.com/adobe/aem-core-forms-components

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/formcontainer/.content.xml [hunk 2]
```diff
@@ -5,4 +5,4 @@
     jcr:title="Adaptive Form Container"
     fd:version="2.1"
     sling:resourceSuperType="core/fd/components/form/container/v2/container"
-    componentGroup=".${appTitle} - Adaptive Form"/>
+    componentGroup="${appTitle} - Adaptive Form"/>
```
Comment (deepprakash345) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/formcontainer/.content.xml:8]: @barshat7 , is this '.' removed intentionally

Comment (barshat7) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/formcontainer/.content.xml:8]: yes, we will need to unhide to support forms in site

**Review by ujjwal5 — APPROVED** [review]
LGTM

**Review by rismehta — APPROVED** [review]
LGTM

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
           jcr:primaryType="cq:Component"
-          jcr:title="AEM Forms Container"
+          jcr:title="Adaptive Form - Embed"
           sling:resourceSuperType="core/fd/components/aemform/v1/aemform"
```
Comment (ujjwal5) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml:5]: @barshat7 we should use latest version(v2) for aemform in archetype. 
cc: @rismehta

**Pull request merged**